bootstrap

```diff
@@ -45,6 +45,16 @@ bootstrap () {
     for a in m4/sage_spkg_versions.m4 m4/sage_spkg_versions_toml.m4; do
         echo 'changequote(`>>>'"'"', `<<<'"')dnl" >> $a
     done
+
+    for pkgname in cypari cysignals cython gmpy2 jupyter_core memory_allocator numpy pkgconfig pplpy setuptools wheel; do
```

Comment:
what does this list of packages consist of?

Reply:
These packages are required at build time and as of this PR have an `install requires` file that is automatically generated. In the future, we can expand this mechanism and include all python packages (with their version defined in the `project.toml` file).